packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx

```diff
@@ -25,33 +66,276 @@ export const YppLandingView: FC = () => {
+                />
+              </GridItem>
+            </CardImageRow>
+          </CardsWithImagesContainer>
+        </StyledLimitedContainerWidth>
+      </BackgroundContainer>
+
+      {/* nfts */}
+      <BackgroundContainer>
+        <StyledLimitedContainerWidth>
+          <CardsWithImagesContainer>
+            <CardImageRow>
+              <GridItem colStart={{ sm: 2, md: 1, lg: 2 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer positionOnMobile="center">
+                  <CardImage absolute src={videoNfts1} alt="Nft tiles" />
+                  <CardImage dropShadow absolute src={videoNfts2} ref={nftCardImageRef} alt="Single nft tile" />
+                  <CardImage
+                    dropShadow
+                    absolute
+                    src={videoNfts3}
+                    ref={nftContextMenuImageRef}
+                    alt="Context menu of nft tile"
+                  />
+                  <CardImage src={videoNfts4} ref={nftCursorImageRef} alt="Cursor" />
+                </ImageContainer>
+              </GridItem>
+              <GridItem colStart={{ sm: 3, md: 8 }} colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}>
+                <ContentCard
+                  title="Monetise your videos with NFT sales and royalties"
+                  subtitle="Video NFTs"
+                  body="Turn your videos into NFTs and put them up for sale.  Choose between fixed price, open auction, and timed auction. Define royalties to get a cut every time your NFT gets sold again.There you can check your channel status and get instructions on completing all actions to get rewards"
+                />
+              </GridItem>
+            </CardImageRow>
+
+            <CardImageRow>
+              <GridItem colStart={{ sm: 2, md: 6, lg: 6 }} colSpan={{ base: 12, sm: 10, md: 7, lg: 6 }}>
+                <ImageContainer>
+                  <CardImage dropShadow absolute src={crt1} alt="Creator token dashboard" />
+                  <CardImage dropShadow absolute src={crt2} ref={cartTokenImageRef} alt="Creator token holders" />
+                  <CardImage src={crt3} />
+                </ImageContainer>
+              </GridItem>
+              <GridItem
+                colStart={{ sm: 3, md: 1, lg: 2 }}
+                rowStart={{ md: 1 }}
+                colSpan={{ base: 12, sm: 8, md: 5, lg: 4 }}
+              >
+                <ContentCard
+                  title="New ways to engage and get support from your audience"
+                  subtitle="CREATOR TOKENS"
+                  body="Turn viewers into evengelists [?] by letting them hold a share in your channel when they buy your own channel token."
+                />
+              </GridItem>
+            </CardImageRow>
+          </CardsWithImagesContainer>
+        </StyledLimitedContainerWidth>
+      </BackgroundContainer>
+      <BackgroundContainer as="footer" noBackground>
```

Comment:
This element has different padding  from SM  than other `BackgroundContainer` 

![Zrzut ekranu 2022-09-8 o 16 06 27](https://user-images.githubusercontent.com/13706246/189143323-a04f4d73-1c41-46fa-bbdb-38ed00e2b2bf.png)